CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   - `sequence`
 - Added support for registering and calling Stored Procedures with `TABLE` return type.
 - Added support for parameter `length` in `StringType()` to specify the maximum number of characters that can be stored by the column.
+- Added support to run on python 3.9 runtime environment.
 - Added the alias `functions.element_at()` for `functions.get()`.
 - Set default reviewer.
 - Add the alias `Column.contains` for `functions.contains`.
```

Comment:
I can't seem to add comments on the lines below this one for some reason. This comment is about the following lines:

- Fixed a bug when `Dataframe.join_table_function` did not run all the necessary queries to setup join table function when sql simplifier was enabled.
- Added comments to SnowflakeFile and added a API reference page for it.

I suggest this:

- Fixed a bug where the `Dataframe.join_table_function` did not run all of the necessary queries to set up the join table function when SQL simplifier was enabled.
- Added comments to SnowflakeFile and added an API reference page for it.

Reply:
I think the SnowflakeFile point is neither a bugfix nor a new feature. I looked at the PR and it is just documentation improvement. 

@sfc-gh-mabrennan should we add changelog for that?